### PR TITLE
Add download CTA for `fr` and `de` /features/private/ page

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
Marketing is testing out a campaign and they'd like to see what adding a Firefox download CTA to `/features/private/` page will result in

## Issue / Bugzilla link

Fixes https://github.com/mozilla/bedrock/issues/14428

## Testing

- [ ] http://localhost:8000/fr/firefox/features/private/
- [ ] http://localhost:8000/de/firefox/features/private/

### CODE REVIEW

**Review by alexgibson — APPROVED** [review]
A couple of small questions/suggestions but otherwise this LGTM r+

One other thing that I guess has already been discussed, but do folks know there's already a Firefox CTA on these pages that gets shown to non-Firefox browsers?

image

File: bedrock/firefox/templates/firefox/features/private.html [hunk 1]
```diff
@@ -16,6 +16,12 @@
 <p>{{ ftl('features-private-firefox-also-protects-your', url='https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=firefox-features') }}</p>
 <p>{{ ftl('features-private-sidenote-we-are-not-big-tech') }}</p>
 
+{% if LANG in ['fr', 'de'] %}
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/firefox/templates/firefox/features/private.html:19]: The marketing team primarily wants to conduct some A/B tests for French and German locales only

File: media/css/firefox/features/article.scss [hunk 2]
```diff
@@ -87,3 +87,10 @@ $image-path: '/media/protocol/img';
         }
     }
 }
+
+.c-firefox-download {
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply media/css/firefox/features/article.scss:94]: oop!

**Comment by reemhamz** [comment]
Leonie confirmed that this is going to be strictly for non-fx users and for the current footer CTA to just be moved above the fold instead of creating an entirely new CTA. I'm just confirming a few things with her before I merge

**Comment by reemhamz** [comment]
@alexgibson I made some minor changes I think are worth re-reviewing if you have the time :) 

Essentially, the original CTA on the page will move above the fold, and we agreed that the image won't be necessary as it pushes down the CTA even further. So now we have a mid-page CTA above-the-fold and no longer have the footer CTA in the page. I've also only made the CTA to show for non-fx users

File: media/js/firefox/features/features-article.es6.js [hunk 3]
```diff
@@ -6,13 +6,18 @@
...
         (footerCta && client.isFirefoxDesktop) ||
         (footerCta && client.isFirefoxAndroid) ||
-        (footerCta && client.isFirefoxiOS)
+        (footerCta && client.isFirefoxiOS) ||
+        (midPageCta && client.isFirefoxDesktop) ||
+        (midPageCta && client.isFirefoxAndroid) ||
+        (midPageCta && client.isFirefoxiOS)
     ) {
         footerCta.parentNode.removeChild(footerCta);
+        midPageCta.parentNode.removeChild(midPageCta);
```
Comment (alexgibson) [inline media/js/firefox/features/features-article.es6.js:21]: Issue: this throws an error in Firefox when viewing a locale that doesn't feature a mid page CTA:

```
Uncaught TypeError: can't access property "parentNode", midPageCta is null
```

Comment (alexgibson) [reply media/js/firefox/features/features-article.es6.js:21]: I think you can also simplify all this logic to:

```javascript
const client = Mozilla.Client;
const footerCta = document.querySelector('.c-feature-footer');
const midPageCta = document.querySelector('.c-firefox-midpage-cta');

if (midPageCta && client.isFirefox) {
    midPageCta.parentNode.removeChild(midPageCta);
}

if (footerCta && client.isFirefox) {
    footerCta.parentNode.removeChild(footerCta);
}
```

**Review by alexgibson — CHANGES_REQUESTED** [review]
Spotted an issue with the JS. The new CTA looks good though!

**Review by alexgibson — APPROVED** [review]
LGTM r+

**Pull request merged**